weasyl/test/db_utils.py

```diff
@@ -24,15 +25,36 @@ def create_api_key(userid, token, description=""):
     add_entity(orm.APIToken(userid=userid, token=token, description=description))
 
 
-def create_user(full_name="", birthday=arrow.get(586162800), config=None, username=None):
+def create_user(full_name="", birthday=arrow.get(586162800), config=None,
+                username=None, password=None, email_addr=None, user_id=None,
+                return_user_definition=False):
```

Comment:
`return_user_definition` isn’t a good argument to have. Use the `username` argument if you need to refer to the username later.